deeppavlov_agent/run_tg.py

```diff
@@ -8,11 +8,17 @@
 from .setup_agent import setup_agent
 
 logger = getLogger(__name__)
-sentry_sdk.init(os.getenv("DP_AGENT_SENTRY_DSN"))  # type: ignore
+sentry_sdk.init(os.getenv('DP_AGENT_SENTRY_DSN'))
 
 
 def run_telegram(cfg: DictConfig):
-    agent, session, workers = setup_agent(cfg.agent)
+    agent, session, workers = setup_agent(
+        cfg.agent.pipeline_config,
+        cfg.agent.db_config,
+        cfg.agent.overwrite_last_chance,
+        cfg.agent.overwrite_timeout,
+        cfg.agent.response_logger,
+    )
```

Comment:
Why? Doesn't fit setup_agent signature.

Reply:
I didn't change that file, it was like that in the origin I forked from. Do you want me to make changes in all such files?